opm/common/utility/numeric/GridUtil.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef GRIDUTIL_H
+#define GRIDUTIL_H
+
+#include <tuple>
+#include <vector>
+#include <opm/common/utility/numeric/VectorUtil.hpp>
+#include <numeric>
+#include <cmath> 
```

Comment:
We generally prefer to include OPM headers before standard headers and, moreover, we prefer to have the standard headers in alphabetical order.  That's mostly to reduce the risk of merge conflicts.  On the other hand, there's nothing here that depends on `VectorUtil.hpp` and in that case you should just drop the `#include` statement for that header.

In general, a header should only `#include` other headers that are strictly needed for client code to compile.  From the current set of declarations you need exactly the following set of standard headers
```C++
<cstddef> // for std::size_t
<tuple>
<vector>
```
All others, if they're needed for the implementation, should go in the `.cpp` file.